src/function.rs

```diff
@@ -590,7 +590,9 @@ impl AttributesData {
             "builtin",
             "cold",
             "convergent",
+            #[cfg(feature = "llvm-15-or-lower")]
             "inaccessiblememonly",
+            #[cfg(feature = "llvm-15-or-lower")]
```

Comment:
I'm not sure about these (or the one below) -- I had to disable them on LLVM > 15 to get some of the tests passing, but I can't find any changesets indicating that these were actually removed.